lib/gui/os/open-external/open-external.js

```diff
@@ -21,9 +21,23 @@
  */
 
 const angular = require('angular');
+const url = require('url');
+
 const MODULE_NAME = 'Etcher.OS.OpenExternal';
 const OSOpenExternal = angular.module(MODULE_NAME, []);
 OSOpenExternal.service('OSOpenExternalService', require('./services/open-external'));
 OSOpenExternal.directive('osOpenExternal', require('./directives/open-external'));
 
+OSOpenExternal.run(function(OSOpenExternalService) {
+  document.addEventListener('click', (event) => {
+    const target = event.target;
+    if (target.tagName === 'A' && angular.isDefined(target.href)) {
```

Comment:
I see. I like the `http` and `https` approach. In order to be on the safe side, I suggest checking if the protocol is *not* `file://` instead. Its probably a small detail but I guess in the future we might want to use ftp links, or application specific protocols?

I'd also appreciate if you can add a comment above the `protocol !== 'file:'` explaining the reason.

Regarding testing, I think its fine. There's no much to cover here anyway.